code/modules/mob/living/living_status_procs.dm

```diff
@@ -413,6 +413,9 @@ STATUS EFFECTS
 /mob/living/proc/Silence(amount)
 	SetSilence(max(amount, AmountSilenced()))
 
+/mob/living/proc/SetAbsoluteSilence(amount)
```

Comment:
I think you might want to keep this proc and add a `/mob/living/proc/AbsoluteSilence(amount)` that calls it with a `max()` and use that where you currently use `SetAbsoluteSilence()`, similar to how `/mob/living/proc/Silence` works.

Reply:
Done